packages/suite-desktop-core/e2e/support/regtest.ts

```diff
@@ -17,6 +17,8 @@ export const generateBlock = () =>
     });
 
 export const waitForCoinjoinBackend = () =>
+    // Todo: refactor to not use async-promise-executor
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
Is it that much work to rewrite these two functions?
If I am not mistaken, this is low hanging fruit, not huge effort to do

```
export const waitForCoinjoinBackend = async  () => {
       const limit = 60;
        const error = '';
        ...
        return; instead of resolve()
        throw; instead of reject()
}
```